src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -467,17 +411,16 @@ public void onWorldTick(TickEvent.WorldTickEvent event)
         }
 
         //  Tick Buildings
-        for (Building building : buildings.values())
+        for (AbstractBuilding building : buildings.values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'building' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)